compiler/rustc_infer/src/traits/util.rs

```diff
@@ -200,6 +200,10 @@ impl<'tcx, O: Elaboratable<'tcx>> Elaborator<'tcx, O> {
         let bound_predicate = elaboratable.predicate().kind();
         match bound_predicate.skip_binder() {
             ty::PredicateKind::Clause(ty::Clause::Trait(data)) => {
+                // Don't elaborate negative trait predicates
```

Comment:
That comment is somewhat redundant with the next two lines. Why don't we need to do anything for negative trait bounds? Is it just because this is the minimal version required for testing?